src/app/design/frontend/base/default/template/factfinder/campaign/advisor.phtml

```diff
@@ -1,6 +1,6 @@
 <?php
 /** @var Omikron_Factfinder_Helper_Data $dataHelper */
-$dataHelper = Mage::helper('factfinder');
+$dataHelper = Mage::helper('factfinder/data');
```

Comment:
this is not necessary, if you skip the what is after the `/` the Magento by convention will load class which name ends with `Data`